kong/clustering/services/sync/rpc.lua

```diff
@@ -388,7 +388,8 @@ local function do_sync()
 end
 
 
-local function sync_handler(premature)
+local sync_handler
+sync_handler = function(premature, try_counter)
```

Comment:
just curious, Is there a special reason not combing these two lines into one, like `local sync_handler = function ...`

Reply:
Just follow the example code of ngx.time.at, https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#ngxtimerat